bower.json

```diff
@@ -17,7 +18,9 @@
     "moment": "~2.10.6",
     "junction": "theleagueof/junction",
     "publiclab-editor": "~0.1.0",
-    "leaflet-d3": "^0.3.8"
+    "leaflet-d3": "^0.3.8",
+    "woofmark": "~4.2.0",
+    "horsey": "3.0.0"
```

Comment:
Yes the javascript modules are indeed complied into PublicLab.editor.js but
the css files aren't so we need these modules. The rich editor template
links the stylesheets of those packages without which the css gets broken.
ᐧ

On Wed, Aug 17, 2016 at 2:29 AM, Jeffrey Warren <notifications@github.com>
wrote:

> In bower.json
> <https://github.com/publiclab/plots2/pull/664#discussion_r75018921>:
>
> > @@ -17,7 +18,9 @@
> >      "moment": "~2.10.6",
> >      "junction": "theleagueof/junction",
> >      "publiclab-editor": "~0.1.0",
> > -    "leaflet-d3": "^0.3.8"
> > +    "leaflet-d3": "^0.3.8",
> > +    "woofmark": "~4.2.0",
> > +    "horsey": "3.0.0"
>
> These are actually compiled into publiclab-editor using browserify, so
> they're not needed in this bower.json file.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/plots2/pull/664/files/31d538c99234a755f3d9868bd56022d400c0e371#r75018921>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKACR9950kpTkYRKWqAeVf3h6nfeqVhJks5qgiSfgaJpZM4JlHKF>
> .
>